spec/UseAtomSpec.php

```diff
@@ -15,8 +15,9 @@
 
     describe("::register()", function() {
         it('should register actions correctly', function() {
-            \WP_Mock::expectActionAdded('init', [$this->useAtom, 'init']);
-            \WP_Mock::expectActionAdded('wp_head', [$this->useAtom, 'wpHead']);
+            allow('add_action')->tobeCalled()->with('init', [$this->useAtom, 'init']);
+            allow('add_action')->tobeCalled()->with('wp_head', [$this->useAtom, 'wpHead']);
```

Comment:
So you need some assertions here, and the `allow()` can be quite simple, because you want to test what is happening to `add_action` you don't want to mock it out completely. 

This test could be something like this:

```php
    describe("::register()", function() {
        it('should register actions correctly', function() {
            allow('add_action')->toBeCalled();
            expect('add_action')->toBeCalled()->times(2);
            expect('add_action')->toBeCalled()->once()->with('init', [$this->useAtom, 'init']);
            expect('add_action')->toBeCalled()->once()->with('wp_head', [$this->useAtom, 'wpHead']);

            $this->useAtom->register();
        });
    });
```

and also you have some other `WP_Mock` boilerplate in the file that needs to be removed